core/src/main/java/jenkins/util/AntClassLoader.java

```diff
@@ -1070,27 +1071,36 @@ protected synchronized Class loadClass(String classname, boolean resolve)
         if (theClass != null) {
             return theClass;
         }
-        if (isParentFirst(classname)) {
-            try {
-                theClass = findBaseClass(classname);
-                log("Class " + classname + " loaded from parent loader " + "(parentFirst)",
-                        Project.MSG_DEBUG);
-            } catch (ClassNotFoundException cnfe) {
-                theClass = findClass(classname);
-                log("Class " + classname + " loaded from ant loader " + "(parentFirst)",
-                        Project.MSG_DEBUG);
-            }
-        } else {
-            try {
-                theClass = findClass(classname);
-                log("Class " + classname + " loaded from ant loader", Project.MSG_DEBUG);
-            } catch (ClassNotFoundException cnfe) {
-                if (ignoreBase) {
-                    throw cnfe;
+
+        //Surround the former logic with a try-catch to report missing class exceptions via Java11 telemetry
+        try {
+            if (isParentFirst(classname)) {
+                try {
+                    theClass = findBaseClass(classname);
+                    log("Class " + classname + " loaded from parent loader " + "(parentFirst)",
```

Comment:
These string concatenations do not look to be necessary. They will happen on any debug level

Reply:
It was already there. Didn't touch to avoid adding noise to the review.